frontends/web/src/api/subscribe.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiSubscribe, Event, Unsubscribe } from '../utils/event';
+import { apiGet } from '../utils/request';
+
+export function subscribeEndpoint(endpoint: string, cb): Unsubscribe {
+    return apiSubscribe(endpoint, (event: Event) => {
+        switch (event.action) {
+        case 'replace':
+            cb(event.object);
+            break;
+        // not used?, deprecate
+        // case 'prepend':
+        //     this.setState(state => ({ [key]: [event.object, ...state[key] as any] } as any));
+        //     break;
+        // case 'append':
+        //     this.setState(state => ({ [key]: [...state[key] as any, event.object] } as any));
+        //     break;
+        // case 'remove':
+        //     this.setState(state => ({ [key]: (state[key] as any).filter((item: any) => !equal(item, event.object)) } as any));
+        //     break;
```

Comment:
TODO: confirm that 'prepend', 'append' and 'remove' hasn't been used.

Reply:
So far searching for `observable.Event{` I only see `action.Replace` and `action.Reload`, I am not 100% sure if I miss something but it looks like prepend, append, remove could be dropped.

Reply:
removed also removed in the backend